salt/modules/cmdmod.py

```diff
@@ -2619,13 +2619,27 @@ def powershell(cmd,
         saltenv='base',
         use_vt=False,
         password=None,
+        depth=None,
         **kwargs):
     '''
-    Execute the passed PowerShell command and return the output as a string.
+    Execute the passed PowerShell command and return the output as a dictionary.
```

Comment:
Disregard, I wrongly thought that we were returning the result of the ``run()`` command.